portfolio/src/main/webapp/script-home.js

```diff
@@ -48,15 +48,24 @@ function main() {
         function adjustForwardOne() {
           slideShowGallery.adjustSlideManual(ADJUST_FORWARD); 
     }
+    // Fetches the pre-made comment list from the servlet and makes 
+    // a list for each comment.
+    fetch('/comments').then(response => response.json()).then((comments) => {
+      const /** ?HTMLCollection */commentContainer =
+            document.getElementById('comments-container');
+      //Iterates through each individual comment/item in the comments json.
```

Comment:
Remove this comment. For each logic is standard in every industry language. No need to explain it